tofu/spectro/_rockingcurve.py

```diff
@@ -15,6 +15,7 @@
 
 # tofu
 from tofu.version import __version__
+from . import _rockingcurve_def as _rockingcurve_def
```

Comment:
_rockingcurve_def does not exist yet (not commited => error

I'll comment this line for this PR, you'll have to un-comment it for the next PR